__tests__/interpolate/Interpolate_transform_test.re

```diff
@@ -0,0 +1,153 @@
+open Jest;
+
+let it = test;
+
+describe("Interpolate_transform", () => {
+  describe("parseTransform", () => {
+    it("should parse a transform into its transform property and value", () =>
+      Expect.(
+        expect(Interpolate_transform.parseTransform("translateX(20px)"))
+        |> toEqual(
+             Interpolate_transform.{
+               transformProperty: Js.Nullable.return("translateX"),
+               transform: "20px",
+             },
+           )
+      )
+    );
+
+    it(
+      "should handle translate, scale, rotate, skew, and perspective in one dimensions",
+      () => {
+        let translate =
+          Interpolate_transform.parseTransform("translateY(20px)");
+        let scale = Interpolate_transform.parseTransform("scaleZ(0.5)");
+        let rotate = Interpolate_transform.parseTransform("rotateX(0.5turn)");
+        let skew = Interpolate_transform.parseTransform("skewZ(1.5rad)");
+        let perspective =
+          Interpolate_transform.parseTransform("perspective(10px)");
+        Expect.(
+          expect((translate, scale, rotate, skew, perspective))
+          |> toEqual((
+               Interpolate_transform.{
+                 transformProperty: Js.Nullable.return("translateY"),
+                 transform: "20px",
+               },
+               Interpolate_transform.{
+                 transformProperty: Js.Nullable.return("scaleZ"),
+                 transform: "0.5",
+               },
+               Interpolate_transform.{
+                 transformProperty: Js.Nullable.return("rotateX"),
+                 transform: "0.5turn",
+               },
+               Interpolate_transform.{
+                 transformProperty: Js.Nullable.return("skewZ"),
+                 transform: "1.5rad",
+               },
+               Interpolate_transform.{
+                 transformProperty: Js.Nullable.return("perspective"),
+                 transform: "10px",
+               },
+             ))
+        );
```

Comment:
Since `bs-jest` allows at max one assertion per test, the canonical way to test a lot of things at once is to place them all in a tuple. In this case, a five-dimensional tuple (it has five elements)!